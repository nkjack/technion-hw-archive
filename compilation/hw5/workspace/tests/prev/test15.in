void maxTestsCheckTruncation() {
	int k = 150b + 150b;
	printi(k);
}

void main() {
     maxTestsCheckTruncation();
     print("\nI hate this course\n");
}
