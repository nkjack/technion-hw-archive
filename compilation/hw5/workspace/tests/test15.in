void TestFunc() {
	int t = 26b + 24b + 39 - 41b - 225b;
	printi(t);
	print("\n");
	int s = 126b + 24b + 13b - 141b - 25b;
	printi(s);
	print("\n");
}

void foo(int x,int y,bool flag)
@pre((x > 0 or x < 10) and (x + y) < 20)
@pre(x < 5 or not flag)
@pre(((x-x)+(y-y))==0)
{
	TestFunc();
}

void main() {
     foo(1,14,false);
     print("GOOD TEST\n");
}
