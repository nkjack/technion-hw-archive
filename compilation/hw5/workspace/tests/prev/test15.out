Loaded: ./exceptions.s
44
I hate this course
